semgrep/tests/e2e/snapshots/test_check/test_multiple_configs_different_origins/results.json

```diff
@@ -1,48 +1,6 @@
 {
   "errors": [],
   "results": [
-    {
```

Comment:
I think the new behavior for deduping is better

Reply:
This one is a bug since they have different check_ids